packages/smart-wallet/test/devices.js

```diff
@@ -1,6 +1,6 @@
 import bundleCentralSupply from '@agoric/vats/bundles/bundle-centralSupply.js';
 import bundleMintHolder from '@agoric/vats/bundles/bundle-mintHolder.js';
-import bundleWalletFactory from '@agoric/vats/bundles/bundle-legacy-walletFactory.js';
+import bundleWalletFactory from '@agoric/vats/bundles/bundle-walletFactory.js';
```

Comment:
We have a `yarn build` for this package now that builds `../bundles/bundle-walletFactory.js`.

Consider using that instead, so while making changes to this package you don't have to build all vats.

Reply:
I lean toward: you can assume `yarn build` is up-to-date for _other_ packages, but changes within a package should get bundled at test-time within a package. So this one is a little awkward: it depends on `yarn build` in `vats` whenever a change to the walletFactory contract in _this_ package is changed.

hm. not sure I have a concrete suggestion.

Reply:
by that principle shouldn't this be as suggested?
```suggestion
import bundleWalletFactory from '../bundles/bundle-walletFactory.js';
```

Reply:
yes, that's good, provided the `test.before()` auto-updates that bundle.